libs/ui-components/src/components/Device/DeviceDetails/DeviceDetails.tsx

```diff
@@ -95,15 +97,47 @@ const DeviceDetails = () => {
             </DetailsPageCardBody>
           </DetailsPageCard>
         </GridItem>
-        <GridItem md={6}>
+        <GridItem md={12} lg={6}>
           <DetailsPageCard>
             <CardTitle>{t('System status')}</CardTitle>
             <DetailsPageCardBody>
               <DescriptionList columnModifier={{ default: '3Col' }}>
                 <DescriptionListGroup>
-                  <DescriptionListTerm>{t('Device status')}</DescriptionListTerm>
+                  <DescriptionListTerm>
+                    <WithHelperText
+                      content={t('Indicates the overall status of application workloads on the device.')}
+                      ariaLabel={t('Application status')}
+                      showLabel
+                    />
+                  </DescriptionListTerm>
                   <DescriptionListDescription>
-                    <DeviceStatus device={device} />
+                    <ApplicationSummaryStatus status={device?.status?.applications.summary?.status} />
+                  </DescriptionListDescription>
+                </DescriptionListGroup>
+                <DescriptionListGroup>
+                  <DescriptionListTerm>
+                    <WithHelperText
+                      content={t('Indicates the overall status of the device hardware and operating system.')}
+                      ariaLabel={t('Device status')}
+                      showLabel
+                    />{' '}
+                  </DescriptionListTerm>
+                  <DescriptionListDescription>
+                    <DeviceStatus deviceStatus={device?.status} />
+                  </DescriptionListDescription>
+                </DescriptionListGroup>
+                <DescriptionListGroup>
+                  <DescriptionListTerm>
+                    <WithHelperText
+                      content={t(
+                        'Indicates whether a system is running the latest target configuration or is updating towards it.',
+                      )}
+                      ariaLabel={t('Update status')}
+                      showLabel
+                    />
+                  </DescriptionListTerm>
+                  <DescriptionListDescription>
+                    <DeviceStatus deviceStatus={device?.status} />
```

Comment:
This should be a new component `SystemUpdateStatus`, I'll add it in the follow-up tasks.